frontend/js/require.config.js

```diff
@@ -35,6 +36,10 @@ require.config({
     waitSeconds: 10,
 
     shim: {
+        "Hls": {
+            deps: ['require'], 
```

Comment:
Can you explain this to me?

Reply:
omes from the MEJS documentation: https://github.com/mediaelement/mediaelement/blob/master/docs/usage.md#requirejs